### PR TITLE
refactor(governance): complete wave6 retirement closeout

### PR DESCRIPTION
**Pull request by jamesryancooper**
Policy reference: `.octon/framework/agency/practices/pull-request-standards.md`
Reminder: Run `@codex review`.

## What

Complete Wave 6 retirement and closeout for the unified execution constitution.
This removes the remaining live transitional execution shims and aligns the
constitutional, runtime, validator, generated, and GitHub automation surfaces
to the final model.

## Why

Wave 6 was implemented, but final confirmation exposed one remaining gap:
durable Wave 6 evidence still embedded the implementing proposal path, so the
proposal was not fully proposal-independent. This PR closes that gap, records
final closeout evidence, and leaves only the final constitutional execution
model live.

No-Issue: internal wave6 closeout

## How

Retire live mission-only and host-shaped approval shims, promote constitutional
families from transitional to active status, remove label/env-backed approval
paths from runtime and GitHub workflows, refresh generated/read-model outputs,
and record the cutover in durable cognition/evidence surfaces. The implementing
proposal is promoted to `implemented`, but not archived in this PR because the
Wave 6 scope explicitly avoided mutating proposal resources/docs beyond the
status promotion.

## Profile Selection Receipt

- Semantic version source(s): `/.octon/octon.yml`
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `transitional`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): constitutional/runtime/workflow/validator cutover had to land together; migration indexes and generated/read-model outputs needed refresh; rollback is one-branch revert; blast radius touched governance, runtime, GitHub automation, and evidence surfaces; hidden host/env approval had to be eliminated fail-closed
- Tie-break status: `transitional` selected for coordinated cutover, even though the resulting state contains no ongoing coexistence window
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): rationale: retire the remaining shims in one coordinated cutover; risks: workflow/runtime drift, stale mission contract fields, premature proposal closeout; owner: `Octon governance`; target_removal_date: `2026-03-28`

## Implementation Plan

- Workstreams and scope:
  - promote constitutional family and obligation status to active
  - retire mission-only execution metadata from active mission/objective surfaces
  - remove label/env-based approval and waiver logic from runtime and GitHub workflows
  - refresh generated cognition/effective/proposal projections and publication receipts
  - record Wave 6 ADR, migration plan, evidence bundle, and continuity updates
- Public interfaces/types/contracts affected:
  - constitutional families under `framework/constitution/**`
  - mission schema and mission registry
  - GitHub autonomy workflows and agency policy docs
  - engine authorization behavior for approval-required execution
- Test scenarios:
  - targeted Wave 1/3/4/5 validators
  - execution governance validator and kernel exact tests
  - alignment-check with `harness,mission-autonomy,agency`
  - proposal validation against promotion targets
- Assumptions/defaults:
  - proposal archival is deferred to a later turn because proposal resource/doc mutation was intentionally constrained

## Impact Map (code, tests, docs, contracts)

- Code/runtime:
  - `.octon/framework/engine/runtime/crates/kernel/src/authorization.rs`
  - `.github/workflows/{pr-autonomy-policy.yml,pr-auto-merge.yml,pr-triage.yml,pr-stale-close.yml,pr-clean-state-enforcer.yml,ai-review-gate.yml}`
- Tests/validators:
  - `.octon/framework/assurance/runtime/_ops/scripts/{validate-objective-binding-cutover.sh,validate-runtime-lifecycle-normalization.sh,validate-assurance-disclosure-expansion.sh,validate-wave5-agency-adapter-hardening.sh,validate-execution-governance.sh,validate-mission-runtime-contracts.sh}`
  - `.octon/framework/assurance/runtime/_ops/tests/test-authority-control-tooling.sh`
- Docs/contracts:
  - `.octon/framework/constitution/**`
  - `.octon/instance/bootstrap/{OBJECTIVE.md,START.md}`
  - `.octon/instance/orchestration/missions/**`
  - `.octon/framework/agency/practices/**`
- Generated/read models/state:
  - `.octon/generated/{cognition,effective,proposals}/**`
  - `.octon/state/evidence/migration/2026-03-28-wave6-retirement-cutover/**`
  - `.octon/state/evidence/validation/publication/{capabilities,extensions}/**`

## Compliance Receipt

- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations

- Proposal archival deferred. Owner: Octon maintainers. Resolution: archive in a dedicated follow-up once proposal resource/doc mutation is allowed.

## Tradeoffs

- High-impact PRs now stay manual-lane only instead of using a label-backed shortcut.
- Publication receipts and generated effective artifacts were refreshed, which increases diff size but keeps state consistent with the final model.
- Historical migration evidence still mentions the retired transitional model as lineage only.

## Testing

- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-objective-binding-cutover.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-runtime-lifecycle-normalization.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-assurance-disclosure-expansion.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-wave5-agency-adapter-hardening.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-execution-governance.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-mission-runtime-contracts.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-harness-structure.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-architecture-conformance.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-continuity-memory.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/alignment-check.sh --profile harness,mission-autonomy,agency`
- `bash .octon/framework/assurance/runtime/_ops/tests/test-authority-control-tooling.sh`
- `cargo test --manifest-path .octon/framework/engine/runtime/crates/Cargo.toml -p octon_kernel authorization::tests::approval_required_autonomous_request_returns_stage_only_without_human_approval -- --exact`
- `cargo test --manifest-path .octon/framework/engine/runtime/crates/Cargo.toml -p octon_kernel authorization::tests::authority_projection_serializes_ref_and_accepts_legacy_alias -- --exact`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-proposal-standard.sh --package .octon/inputs/exploratory/proposals/architecture/fully-unified-execution-constitution-for-governed-autonomous-work`

## Rollout

- No feature flag rollout.
- Land as a single governance/runtime closeout branch so contracts, validators,
  workflows, generated outputs, and evidence stay consistent.
- Proposal status is promoted to `implemented` in this PR; archive in follow-up.

## Convivial Purpose Check

- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric

- Risk class: [ ] Trivial [ ] Low [x] Medium [ ] High
- Rollback plan: revert the Wave 6 branch as one change set
- Rollback handle: `git revert <squash-commit>` or PR revert
- Flags changed (name, owner, expiry, rollout): `n/a`
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model

- OpenAPI/JSON-Schema changes: mission charter schema removes transitional execution metadata; constitutional family/obligation semantics updated
- Threat model update/link: host/env approval no longer materializes authority; labels/checks stay non-authoritative projections

## Observability and Performance

- Traces/logs/metrics for changed flows: existing execution/run evidence and publication receipts remain canonical
- Representative traces for high risk changes: validator and publication receipts captured under `.octon/state/evidence/**`
- Performance or bundle impact: none material; diff size increases from regenerated state/publication artifacts only

## License and Provenance

- New dependencies and licenses: none
- Generated code/templates provenance notes: cognition/effective/proposal outputs and publication receipts regenerated locally from canonical manifests/scripts

## Checklist

- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [x] All review conversations resolved

## Screenshots / Notes

- No visual changes.
- Wave 6 is complete in the live durable model. Proposal archival is the only deferred lifecycle action.

### CODE REVIEW

File: .octon/framework/agency/practices/git-autonomy-playbook.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ Behavior:
 
 When touching high-impact paths (for example `.github/` or governance paths):
 
-1. Add `accept:human` before merge.
+1. Leave the PR in the manual lane and complete ordinary human review.
 2. Keep PR focused and explicitly document risk/rollback.
 3. Use `git-pr-ship.sh --accept-human` to preserve metadata-level check-in.
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/agency/practices/git-autonomy-playbook.md]: Addressed in `603936a75`.

The playbook no longer references `git-pr-ship.sh --accept-human`; it now points to the current manual-lane guidance and the shipped `git-pr-ship.sh` interface.
